source/DistanceAndDirection/ArcMapAddinDistanceAndDirection/ArcMapAddinDistanceAndDirection/MapPointTool.cs

```diff
@@ -79,6 +80,8 @@ protected override void OnMouseMove(MouseEventArgs arg)
         {
             try
             {
+                IMouseCursor cursor = new MouseCursorClass();
```

Comment:
It looks like there is also a cursor property on an addin tool (to set OnActivate?): http://help.arcgis.com/en/sdk/10.0/arcobjects_net/componenthelp/index.html#/Tool_Members/001v0000008w000000/